tests/test_jx/test_deep_ops.py

```diff
@@ -820,9 +820,7 @@ def test_aggs_on_parent_and_child(self):
                     {"o": 1, "v": NULL, "s": 0},
                     {"o": 2, "v": "b", "s": 1},
                     {"o": 2, "v": "c", "s": 0},
-                    {"o": 2, "v": NULL, "s": 1},
-                    {"o": NULL, "v": "b", "s": 0},
-                    {"o": NULL, "v": "c", "s": 0}
+                    {"o": 2, "v": NULL, "s": 1}
```

Comment:
This and the previous cases are different. Here, we are not inserting any document without `"o"`property, so we don't expect the result to include `NULL`'s. But in the previous argument, documents with `Null` was inserted(with property absent in current document), and hence the data was not actually redundant as it is in this case. I will add a test to show this concern of mine.